sdk/springappdiscovery/azure-resourcemanager-springappdiscovery/src/main/java/com/azure/resourcemanager/springappdiscovery/SpringAppsDiscoveryManager.java

```diff
@@ -0,0 +1,343 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.resourcemanager.springappdiscovery;
+
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.http.HttpClient;
+import com.azure.core.http.HttpPipeline;
+import com.azure.core.http.HttpPipelineBuilder;
+import com.azure.core.http.HttpPipelinePosition;
+import com.azure.core.http.policy.AddDatePolicy;
+import com.azure.core.http.policy.AddHeadersFromContextPolicy;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.http.policy.HttpLoggingPolicy;
+import com.azure.core.http.policy.HttpPipelinePolicy;
+import com.azure.core.http.policy.HttpPolicyProviders;
+import com.azure.core.http.policy.RequestIdPolicy;
+import com.azure.core.http.policy.RetryOptions;
+import com.azure.core.http.policy.RetryPolicy;
+import com.azure.core.http.policy.UserAgentPolicy;
+import com.azure.core.management.http.policy.ArmChallengeAuthenticationPolicy;
+import com.azure.core.management.profile.AzureProfile;
+import com.azure.core.util.Configuration;
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.resourcemanager.springappdiscovery.fluent.SpringAppsDiscoveryClient;
+import com.azure.resourcemanager.springappdiscovery.implementation.ErrorSummariesImpl;
+import com.azure.resourcemanager.springappdiscovery.implementation.OperationsImpl;
+import com.azure.resourcemanager.springappdiscovery.implementation.SpringAppsDiscoveryClientBuilder;
+import com.azure.resourcemanager.springappdiscovery.implementation.SpringbootappsImpl;
+import com.azure.resourcemanager.springappdiscovery.implementation.SpringbootserversImpl;
+import com.azure.resourcemanager.springappdiscovery.implementation.SpringbootsitesImpl;
+import com.azure.resourcemanager.springappdiscovery.implementation.SummariesImpl;
+import com.azure.resourcemanager.springappdiscovery.models.ErrorSummaries;
+import com.azure.resourcemanager.springappdiscovery.models.Operations;
+import com.azure.resourcemanager.springappdiscovery.models.Springbootapps;
+import com.azure.resourcemanager.springappdiscovery.models.Springbootservers;
+import com.azure.resourcemanager.springappdiscovery.models.Springbootsites;
+import com.azure.resourcemanager.springappdiscovery.models.Summaries;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+/**
+ * Entry point to SpringAppsDiscoveryManager.
+ * Spring App Discovery Client.
+ */
+public final class SpringAppsDiscoveryManager {
```

Comment:
package name is `springappdiscovery`, while manager name is `SpringAppsDiscoveryManager`, `Apps`(extra `s` ), do we want to call it `SpringAppDiscoveryManager`

Reply:
Service wrote that themselves https://github.com/Azure/azure-rest-api-specs/pull/26365/files#diff-9d93041b337456d8fcf32c76e968862025948ce3c515f9eb4e06cce6933485ff

But seems wrong, agree that `App` without `s` is better. That is also what python uses. Will update.

Reply:
spec PR https://github.com/Azure/azure-rest-api-specs/pull/27494